packages/core/src/rum/instrumentation/DdRumErrorTracking.tsx

```diff
@@ -66,8 +66,7 @@ export class DdRumErrorTracking {
         }
     }
 
-    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    static onGlobalError(error: any, isFatal?: boolean): void {
+    static onGlobalError = (error: any, isFatal?: boolean): void => {
```

Comment:
When creating a `function(){}` instead of arrow function `function = () => {}`, `this` references by default the caller, not the callee.
Using an arrow function ensures `this` is the callee, which is what we want here.